native/cocos/bindings/manual/jsb_module_register.cpp

```diff
@@ -79,10 +79,11 @@
     #include "cocos/bindings/manual/JavaScriptJavaBridge.h"
 #endif
 
-#if (CC_PLATFORM == CC_PLATFORM_IOS || CC_PLATFORM == CC_PLATFORM_ANDROID || CC_PLATFORM == CC_PLATFORM_OHOS)
-
-    #if CC_USE_VIDEO
-        #include "cocos/bindings/auto/jsb_video_auto.h"
+#if (CC_PLATFORM == CC_PLATFORM_IOS || CC_PLATFORM == CC_PLATFORM_ANDROID || CC_PLATFORM == CC_PLATFORM_OHOS || CC_PLATFORM == CC_PLATFORM_OPENHARMONY)
+    #if(CC_PLATFORM != CC_PLATFORM_OPENHARMONY)
```

Comment:
It is wired. Why not just don't include `CC_PLATFORM_OPENHARMONY ` in outer `#if`?

Reply:
This macro contains the webview and videoplay